### PR TITLE
Add LLM_API_KEY and LLM_MODEL to README.md

### PR DESCRIPTION
**Pull request by hwaking**
add llm keys in docker run cmd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,13 +58,22 @@ To start the app, run these commands, replacing `$(pwd)/workspace` with the dire
...
+#export LLM_BASE_URL="https://api.groq.com/openai/v1"
+
 ```
 
 ```bash
 docker run -it \
     --pull=always \
     -e SANDBOX_USER_ID=$(id -u) \
     -e PERSIST_SANDBOX="true" \
+    -e LLM_API_KEY \
```
Comment (tobitege) [inline README.md:75]: env vars are missing

File: README.md [hunk 2]
```diff
@@ -58,13 +58,22 @@ To start the app, run these commands, replacing `$(pwd)/workspace` with the dire
 ```bash
 # The directory you want OpenDevin to work with. MUST be an absolute path!
 export WORKSPACE_BASE=$(pwd)/workspace;
+
+# Your OpenAI API key, or any other LLM API key
+#export LLM_API_KEY="sk-..."
+export LLM_API_KEY="sk-"
+export LLM_MODEL="groq/llama3-70b-8192"
```
Comment (yufansong) [inline README.md:65]: Sugesst default to gpt 4o
```suggestion
export LLM_MODEL="azure/gpt-4o"
```

Comment (rbren) [reply README.md:65]: These should not  be in the README, they are set via the UI

**Review by rbren — CHANGES_REQUESTED** [review]
@hwaking I'm curious what made you add this. These things should be set via the UI, not the command line

**Comment by li-boxuan** [comment]
I agree with @rbren . Setting them from cmdline is handy for dev, but not something we should teach users.

**Comment by tobitege** [comment]
> I agree with @rbren . Setting them from cmdline is handy for dev, but not something we should teach users.

OpenDevin, as most other Python-based repos, actually require a technical level of understanding that imho requires a "user" to be very close of being a "developer".
With "not teaching" means making it more work for devs to find the useful tidbits and maybe just add a short note like "these env vars are normally set in the UI and optional here" or similar.

**Comment by li-boxuan** [comment]
> With "not teaching" means making it more work for devs to find the useful tidbits and maybe just add a short note like "these env vars are normally set in the UI and optional here" or similar.

We do have dev doc, that would be a more appropriate place.

**Comment by neubig** [comment]
Thanks for the contribution @hwaking ! But as @rbren pointed out, these variables are (for now) set via the "settings" pane in the interface. I'm going to close this PR, but if the documentation about how setting them through the interface is not clear enough, we can open another PR to try to fix this.

**Pull request closed without merging**